lib/diagnostic.js

```diff
@@ -16,29 +16,45 @@
 // Attributes: name, commissionRate
 // Methods: sell, call
 
-// Answer Starts Here
+// Description: Calendar app to keep track of things.
+// Entity: User
+// Attributes: name, email, address
+// Methods: Output of what the user did on a specific date.
 //
-// Answer Ends Here
+// Entity: Date
+// Attributes: date,
+// Method: What needs to be done on each date, update to do list.
+
 
 // 2. Create an object literal that lines up with the following `owner` entity description.
 // Store it in the variable below. Be sure to give it reasonable values for each
 // of its properties.
 //
 // Entity: Owner
 // Attributes: name, address, numberOfPets
+const Owner = {
+    name: 'Aneela',
+    address: '1 Devonshire Pl',
+    numberOfPets: 0,
+};
+console.log(Owner.name);
 
-// Replace null with your answer
-let owner = null;
 
 // 3. Create an object literal that lines up with the following `pet` entity description.
 // Store it in the variable `pet`, below.
 
 // Entity: Pet
 // Attributes: name, species, breed, vocalization
 // Methods: vocalize
+let Pet = {
+  name: 'Ted',
+  species: 'dog',
+  breed: 'dalmation',
+  vocalization: 'loud',
+  }
```

Comment:
```js
vocalize: function () {
    return this.vocalization;
  }
};
```

You just need to add a vocalize method that returns the vocalization of this object.